src/assets/assets.controller.ts

```diff
@@ -69,9 +82,20 @@ export class AssetsController {
       const transaction = await this.transactionsService.findOrThrow(
         asset.created_transaction_id,
       );
+
```

Comment:
Lame that we have to search through blocks just to get the timestamp, but also doesn't seem worth it to write a backfill/update to transaction model for timestamp.

Reply:
Yeah, I'll do that in a subsequent change without us needing to do a reset